### PR TITLE
Playground refresh tutorial updates

### PR DESCRIPTION
**Pull request by j-mueller**
Pre-submit checklist:
- Branch
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [ ] `$(nix-build default.nix -A plutus.updateMaterialized)` to update the materialized Nix files
   - [ ] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
   - [ ] `$(nix-shell shell.nix --run fix-stylish-haskell)` to fix any formatting issues
- If you changed any Purescript files:
   - [ ] `$(nix-shell shell.nix --run fix-purty)` to fix any formatting issues

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: doc/tutorials/plutus-playground.rst [hunk 1]
```diff
@@ -10,54 +10,64 @@ The Plutus Playground consists of a code editor and a simulator.
...
 
-We can use the simulator to define complex scenarios with multiple agents trading and communicating over the Cardano blockchain.
-For the "Hello, World" app however we don't need to change anything in the simulator.
-We can click "Evaluate" right away.
+You can use the simulator to define complex scenarios with multiple agents trading and communicating over Cardano.
+For the "Hello, World" app however you don't need to change anything in the simulator.
+You can click **Evaluate** right away.
 This sends the simulation to the server, alongside the source code of our app.
-When the server is done running the simulation, the transactions pane opens, showing the outcome of the simulation.
+When the server is finished running the simulation, the transactions pane is displayedm which presents the outcome of the simulation.
```
Comment (gilligan) [inline doc/tutorials/plutus-playground.rst:61]:
```suggestion
When the server is finished running the simulation, the transactions pane is displayed, which presents the outcome of the simulation.
```

File: doc/tutorials/plutus-playground.rst [hunk 2]
```diff
@@ -10,54 +10,64 @@ The Plutus Playground consists of a code editor and a simulator.
...
 
-We can use the simulator to define complex scenarios with multiple agents trading and communicating over the Cardano blockchain.
-For the "Hello, World" app however we don't need to change anything in the simulator.
-We can click "Evaluate" right away.
+You can use the simulator to define complex scenarios with multiple agents trading and communicating over Cardano.
+For the "Hello, World" app however you don't need to change anything in the simulator.
+You can click **Evaluate** right away.
 This sends the simulation to the server, alongside the source code of our app.
-When the server is done running the simulation, the transactions pane opens, showing the outcome of the simulation.
+When the server is finished running the simulation, the transactions pane is displayed, which presents the outcome of the simulation.
```
Comment (merivale) [inline doc/tutorials/plutus-playground.rst:61]:
```suggestion
When the server has finished running the simulation, the transactions pane is displayed, which presents the outcome of the simulation.
```

File: doc/tutorials/plutus-playground.rst [hunk 3]
```diff
@@ -68,98 +78,106 @@ The blockchain section shows all the transactions that were produced during the
 Each simulation is run on a fresh blockchain.
 The "Hello, World" simulation only results in a single transaction.
 This is the initial transaction that distributes some funds to the simulated agents.
-We can click the box that says "Slot 0, Tx 0" to examine the transaction.
+You can click the box that says "Slot 0, Tx 0" to examine the transaction.
```
Comment (merivale) [inline doc/tutorials/plutus-playground.rst:81]: I just merged a PR that makes the first transaction preselected by default.

File: doc/tutorials/plutus-playground.rst [hunk 4]
```diff
@@ -68,98 +78,106 @@ The blockchain section shows all the transactions that were produced during the
...
 
-Now we're going to extend the simulation by adding a new wallet and a payment between two wallets.
-Click the close button to get back to the simulator.
+Now you can extend the simulation by adding a new wallet and a payment between two wallets.
+Click the **Close** button to return to the simulator.
 In the "Wallets" section of the simulator there is a box for each simulated wallet.
-Click the "Add Wallet" button to add a new simulated wallet.
-Now click "Pay to Wallet" on wallet 3. In the "Actions" section there are now two entries, a "Wait" action and a "Pay to Wallet" action.
+Click **Add Wallet** to add a new simulated wallet.
+Now click **Pay to Wallet** on wallet 3. In the "Actions" section there are now two entries, a "Wait" action and a "Pay to Wallet" action.
```
Comment (gilligan) [inline doc/tutorials/plutus-playground.rst:120]:
```suggestion
Now click **Pay to Wallet** on wallet 3. In the "Actions" section there are now two entries, an "Add Wait Action" action and a "Pay to Wallet" action.
```

File: doc/tutorials/plutus-playground.rst (hunk 3 above)
Comment (j-mueller) [reply doc/tutorials/plutus-playground.rst]: Good catch!

**Comment by j-mueller** [comment]
This is just missing 1 link, then it's good to go.

**Pull request merged**